bot/pagination.py

```diff
@@ -87,6 +110,41 @@ def add_line(self, line: str = '', *, empty: bool = False) -> None:
             self._current_page.append('')
             self._count += 1
 
+        if remaining_words:
+            self.add_line(remaining_words)
+
+    def _split_remaining_words(self, line: str, max_chars: int) -> t.Tuple[str, t.Optional[str]]:
+        """
+        Internal: split a line into two strings -- reduced_words and remaining_words.
+
+        reduced_words: the remaining words in `line`, after attempting to remove all words that
+            exceed `max_chars` (rounding down to the nearest word boundary).
+
+        remaining_words: the words in `line` which exceed `max_chars`. This value is None if
+            no words could be split from `line`.
+
+        Return a tuple in the format (reduced_words, remaining_words).
+        """
+        reduced_words = []
+        # "(Continued)" is used on a line by itself to indicate the continuation of last page
+        remaining_words = ["(Continued)\n", "---------------\n"]
+        reduced_char_count = 0
+        is_full = False
+
+        for word in line.split(" "):
+            if not is_full:
+                if len(word) + reduced_char_count <= max_chars:
+                    reduced_words.append(word)
+                    reduced_char_count += len(word)
```

Comment:
I'm just realizing that this has a bit of a logical flaw: it doesn't account for the space characters. I'll have to adjust this approach slightly.